package.json

```diff
@@ -51,18 +50,20 @@
     "Alexis Masseron <alexis.masseron@consensys.net>",
     "Christian Tran <christian.tran@consensys.net>",
     "Julien Marchand <julien.marchand@consensys.net>",
-    "Nicolas Maurice <nicolas.maurice@consensys.net>"
+    "Nicolas Maurice <nicolas.maurice@consensys.net>",
+    "Nicolas Massart <nicolas.massart@consensys.net>"
   ],
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/PegaSysEng/orchestrate-node/issues"
   },
-  "homepage": "https://github.com/PegaSysEng/orchestrate-node#readme",
+  "homepage": "https://pegasyseng.github.io/orchestrate-node/",
   "dependencies": {
     "@grpc/grpc-js": "~0.6.15",
     "commander": "~4.1.0",
     "ethers": "~4.0.43",
     "kafkajs": "~1.11.0",
+    "minimist": "^1.2.5",
```

Comment:
can you finx the version to the patch by using `~` instead of `~` ? We don't trust minor updates haha

Reply:
ok, this was written by NPM I did not see that... sorry.

Reply:
Also this is why `npm ci` is cool it's because it will not update automatically. It uses the locked versions only. So if update happens it's on when a dev updates with `npm install` and then it's tested as part of the PR and we can spot the issue beforehand.